devices/lixee.js

```diff
@@ -0,0 +1,697 @@
+/* eslint-disable max-len */
+const exposes = require('../lib/exposes');
+const globalStore = require('../lib/store');
+const {repInterval} = require('../lib/constants');
+const reporting = require('../lib/reporting');
```

Comment:
Import Buffer for fix `Exception while calling fromZigbee converter: Buffer is not defined}`

```suggestion
const reporting = require('../lib/reporting');
const Buffer = require("buffer");
```

Reply:
Thanks. Is weird that in my side I was able to launch it without problems....